numpy/core/fromnumeric.py

```diff
@@ -2724,8 +2731,8 @@ def prod(a, axis=None, dtype=None, out=None, keepdims=np._NoValue, initial=np._N
     raised on overflow.  That means that, on a 32-bit platform:
 
     >>> x = np.array([536870910, 536870910, 536870910, 536870910])
-    >>> np.prod(x)  # random
-    16
+    >>> np.prod(x)
+    6917529010461212688 # may vary
```

Comment:
This is in the context of "this is how it behaves on a 32-bit platform". We should put it back, or maybe just stick `dtype=np.int32` here